lib/remap-functions/src/decode_base64.rs

```diff
@@ -0,0 +1,83 @@
+use remap::prelude::*;
+
+#[derive(Clone, Copy, Debug)]
+pub struct DecodeBase64;
+
+impl Function for DecodeBase64 {
+    fn identifier(&self) -> &'static str {
+        "decode_base64"
+    }
+
+    fn parameters(&self) -> &'static [Parameter] {
+        &[Parameter {
+            keyword: "value",
+            accepts: |v| matches!(v, Value::Bytes(_)),
+            required: true,
+        }]
+    }
+
+    fn compile(&self, mut arguments: ArgumentList) -> Result<Box<dyn Expression>> {
+        let value = arguments.required("value")?.boxed();
+
+        Ok(Box::new(DecodeBase64Fn { value }))
+    }
+}
+
+#[derive(Clone, Debug)]
+struct DecodeBase64Fn {
+    value: Box<dyn Expression>,
+}
+
+impl Expression for DecodeBase64Fn {
+    fn execute(&self, state: &mut state::Program, object: &mut dyn Object) -> Result<Value> {
+        let value = self.value.execute(state, object)?.try_bytes()?;
+
+        match base64::decode(value) {
```

Comment:
Will this handle decoding base64 values lacking padding? Could we add a test for that?

Reply:
Good call. The answer is yes. Test added.